R/functions.R

```diff
@@ -127,39 +127,47 @@ summarizeAtlasExperiment <- function( experimentAccession, atlasExperimentDirect
 		stop( paste( "XML config file", atlasExperimentXMLfile, "does not exist or is not readable." ) )
 	}
     
-    cat( paste( "Reading experiment config from", atlasExperimentXMLfile, "...\n" ) )
+  cat( paste( "Reading experiment config from", atlasExperimentXMLfile, "...\n" ) )
 
 	# Parse the XML file.
 	experimentXMLlist <- parseAtlasConfig( atlasExperimentXMLfile )
 
-    cat( "Finished reading experiment config\n" )
+  cat( "Finished reading experiment config\n" )
 
-	# Get the pipeline code from the experiment accession e.g. MTAB, MEXP
-	pipeline <- gsub( "E-", "", experimentAccession )
-	pipeline <- gsub( "-\\d+", "", pipeline )
-
-	# Filename for SDRF.
-	sdrfBasename <- paste( experimentAccession, ".sdrf.txt", sep="" )
+  #read SDRF file from a shell environment variable if it exists (otherwise it returns "")
+  sdrfPath <- Sys.getenv(x = 'SDRF_PATH', unset = "", names = NA)
```

Comment:
We can drop the x here
```suggestion
  sdrfPath <- Sys.getenv('SDRF_PATH', unset = "", names = NA)
```